jbpm-console-ng-dashboard/jbpm-console-ng-dashboard-client/src/main/java/org/jbpm/dashboard/renderer/client/panel/ProcessDashboardViewImpl.ui.xml

```diff
@@ -38,20 +38,21 @@
     }
 
     .header {
-      padding-left: 10px;
-      padding-bottom: 20px;
+      padding: 15px;
       background-color: white;
     }
 
+    .breadcrumb {
```

Comment:
For the breadcrumb component, PatternFly already has a component https://www.patternfly.org/widgets/#breadcrumbs.
I guess that should be in the j:ProcessBreadCrumb element, so no custom css would be needed in here.